_python/scriptmodified.py

```diff
@@ -0,0 +1,95 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+import numpy as np
+from sqlalchemy import create_engine
+import numpy as np
+import os
+import logging
+
+
+logging.basicConfig(level=logging.INFO)
+
+class WikiParseWar:
+    def __init__(self,url):
+        self.url = url
+        self.soup = None
+        self.df = pd.DataFrame()
+        logging.info(f'DATABASE_URL: {os.getenv("DATABASE_URL")}')
+    
+
+    def getPage(self):
+        try:
+            page = requests.get(self.url)
+            self.soup = BeautifulSoup(page.text,'html')
+        except requests.RequestException as e:
+            logging.error (f'Error fetching page {e}')
+
+    def getTables(self):
```

Comment:
This can have @staticmethod, please change it to
`return self.soup.find_all('table',class_='wikitable')`